src/snac/server/database/SQL.php

```diff
@@ -3852,12 +3858,74 @@ public function insertResource(        $resourceID,
                           $docTypeID,         // 7
                           $entryTypeID,       // 8
                           $link,              // 9
-                          $objectXMLWrap);    // 10
+                          $objectXMLWrap,     // 10
+                          $date,              // 11
+                          $displayEntry,      // 12
+                          $userID);          // 13
         $this->sdb->execute($qq, $execList);
         $this->sdb->deallocate($qq);
         return array($resourceID, $resourceVersion);
     }
 
+
```

Comment:
Can we remove this commented-out function and leave it archived in git?

Reply:
Great idea, I'll do that.